packages/header-component/src/components/dc-header/dc-header.tsx

```diff
@@ -205,7 +205,16 @@ export class Header {
           </nav>
           <div class="session-items">
             {this.user ? (
-              <dc-user-items user={this.user} community={this.community} />
+              <div class="with-user">
+                <slot name="donate">
+                  {this.donateurl && (
```

Comment:
Can we remove this now the fix has landed on Stencil?

Reply:
It has been merge to master on the stenciljs project but it hasn't get released yet @orlando 
https://github.com/ionic-team/stencil/commits/master currently the latest version is 2.3.0 and it doesn't have the fix yet.

Reply:
I have created a [technical debt](https://app.asana.com/0/1195530778765030/1199244186304413) for this one so we can move forward now